level03/level03.cabal

```diff
@@ -109,5 +109,5 @@ test-suite level03-tests
                       , wai == 3.2.*
                       , wai-extra == 3.0.*
                       , hspec == 2.4.*
-                      , hspec-wai == 0.8.*
+                      , hspec-wai == 0.9.*
```

Comment:
It might be better to use `hspec-wai >= 0.8 && < 0.10` here and in other projects.

Reply:
I concur, although for this PR it should be okay. It passes CI with the changes to the versions for the different packages so I'm okay with it as is.

Moving forward the bounds should be defined in terms of an upper and lower bound, to provide a bit more flexibility.

Thoughts ? @endgame